### PR TITLE
Migrate Postfix parser

### PR DESCRIPTION
**Pull request by lvoloshyn-sekoia**

### CODE REVIEW

File: Postfix/postfix/tests/cleanup.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      ],
+      "kind": "event",
+      "type": [
+        "info"
+      ]
+    },
+    "email": {
+      "to": {
+        "address": [
+          "e@exemple.com"
```
Comment (squioc) [inline Postfix/postfix/tests/cleanup.json:19]:
```suggestion
          "john.doe@exemple.com"
```

File: Postfix/postfix/ingest/parser.yml [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+  set_basic_fields:
+    actions:
+      - set:
+          event.kind: "event"
+          event.category: ["email"]
+          event.type: ["info"]
+
+      - set:
+          os.family: "linux"
+          os.platform: "linux"
```
Comment (squioc) [inline Postfix/postfix/ingest/parser.yml:208]: Useless. We can remove these fields.

File: Postfix/postfix/tests/cleanup.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+  "input": {
+    "message": "77EFFC0015: warning: header Content-Disposition: inline; filename=\"image003.jpg\"; size=26055;??creation-date=\"Thu, 12 Sep 2019 12:39:01 GMT\";??modification-date=\"Thu, 12 Sep 2019 12:40:01 GMT\" from mail.outbound.protection.outlook.com[52.100.135.105]; from=<> to=<john.doe@exemple.com> proto=ESMTP helo=<NAM03.outbound.protection.outlook.com>"
+  },
+  "expected": {
+    "message": "77EFFC0015: warning: header Content-Disposition: inline; filename=\"image003.jpg\"; size=26055;??creation-date=\"Thu, 12 Sep 2019 12:39:01 GMT\";??modification-date=\"Thu, 12 Sep 2019 12:40:01 GMT\" from mail.outbound.protection.outlook.com[52.100.135.105]; from=<> to=<john.doe@exemple.com> proto=ESMTP helo=<NAM03.outbound.protection.outlook.com>",
```
Comment (TOUFIKIzakarya) [inline Postfix/postfix/tests/cleanup.json:6]: Can we get the `proto=` in `network.protocol`

File: Postfix/postfix/tests/filename3.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        ]
+      },
+      "to": {
+        "address": [
+          "Pipin.touque@lacomte.net"
+        ]
+      }
+    },
+    "file": {
+      "name": "?iso-8859-2?q?representative_on_migration.pdf?=",
```
Comment (TOUFIKIzakarya) [inline Postfix/postfix/tests/filename3.json:29]: I think the `file.name` should end with `.pdf`

**Review by squioc — CHANGES_REQUESTED** [review]
Thanks for the job.
Some fixes to apply.

File: Postfix/postfix/tests/filename3.json (hunk 4 above)
Comment (lvoloshyn-sekoia) [reply Postfix/postfix/tests/filename3.json:29]: As hideous as it is it, this value was provided in a sample and I was able to create a file with this name. Not sure if we have to truncate this.

**Review by TOUFIKIzakarya — APPROVED** [review]
LGFM

**Review by squioc — APPROVED** [review]
LGTM

**Pull request merged**